packages/ai-core/src/browser/prompttemplate-contribution.ts

```diff
@@ -34,15 +34,15 @@ export const PROMPT_TEMPLATE_EXTENSION = '.prompttemplate';
 export const DISCARD_PROMPT_TEMPLATE_CUSTOMIZATIONS: Command = Command.toLocalizedCommand({
     id: 'theia-ai-prompt-template:discard',
     iconClass: codicon('discard'),
-    category: 'Theia AI Prompt Templates'
+    category: 'AI Prompt Templates'
 }, '', 'theia/ai/core/prompts/category');
 
 // TODO this command is mainly for testing purposes
 export const SHOW_ALL_PROMPTS_COMMAND: Command = Command.toLocalizedCommand({
```

Comment:
We could remove this command alltogether. Can be done in a follow up, but if you don't mind, just remove it.